Website/src/rendering/src/components/FeaturedSpeakers.tsx

```diff
@@ -7,38 +7,46 @@ type Speaker = {
     Name: Field<string>;
     Role: Field<string>;
     Picture: ImageField;
+    Featured: Field<boolean>;
   };
 };
 
 type FeaturedSpeakersProps = ComponentProps & {
   fields: {
-    Speakers: Speaker[];
+    items: Speaker[];
+  };
+  params: {
+    NumberOfSpeakers: string;
   };
 };
 
 const FeaturedSpeakers = (props: FeaturedSpeakersProps): JSX.Element => (
   <div className="item-grid">
     <div className="grid-content">
-      {props.fields.Speakers &&
-        props.fields.Speakers.map((speaker, index) => (
-          <Link
-            key={index}
-            href={'/speakers/' + speaker.fields.Name.value.replace(/ /g, '')}
-            passHref
-          >
-            <a className="grid-item">
-              <Image
-                field={speaker.fields.Picture}
-                alt={speaker.fields.Name?.value}
-                width={265}
-                height={265}
-              />
-              <div className="item-details">
-                <Text tag="p" field={speaker.fields.Name}></Text>
-              </div>
-            </a>
-          </Link>
-        ))}
+      {props.fields.items &&
+        props.fields.items
+          .filter((item) => item.fields.Featured.value === true)
```

Comment:
`item.fields.Featured.value === true` could be simplified to `item.fields.Featured.value`